knimin/tests/test_ag_new_kit.py

```diff
@@ -124,5 +124,17 @@ def test_post(self):
         self.assertIn("SET assigned_on = NOW() WHERE barcode IN ()",
                       response.body)
 
+        # what if tag is None
+        response = self.post('/ag_new_kit/',
+                             {'tag': '',
+                              'projects': ['PROJECT2', 'PROJECT5'],
+                              'swabs': swabs,
+                              'kits': kits,
+                              })
+        self.assertEqual(response.code, 200)
+        kitinfo = loads(response.body)
+        self.assertNotIn('_', kitinfo['kitinfo'][0][0])
```

Comment:
true ... at first. I want to test that there is no prefix (= "tag") set. The kit ID is composed of <tag>_<random> if tag == None than we don't get the _ as well. Thus I think it is the only way to check for the absence of a tag. Or do you see another way?